content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -146,20 +146,6 @@ All code snippets in Windows sections are to be run in a PowerShell environment
 with elevated permissions (Administrator) on the Windows worker node.
 {{< /note >}}
 
```

Comment:
```suggestion

{{< tabs name="tab-windows-kubeadm-runtime-installation" >}}
```
See @sftim's [comment](https://github.com/kubernetes/website/pull/31059/files#r773544210) that the tab starts on line 150